pcdsdevices/qadc.py

```diff
@@ -0,0 +1,147 @@
+from ophyd import Component as Cpt
+from ophyd import Device, EpicsSignal
+
+from .interface import BaseInterface
+from .variety import set_metadata
+
+
+class QadcCommon(BaseInterface, Device):
+    """
+    Common class for Abaco FMC digitizers. Used in Qadc and Qadc134.
+    """
+
+    ch0 = Cpt(EpicsSignal, ":CH0", kind="normal", doc="Input 0 of the ADC")
+    ch1 = Cpt(EpicsSignal, ":CH1", kind="normal", doc="Input 1 of the ADC")
+    ch2 = Cpt(EpicsSignal, ":CH2", kind="normal", doc="Input 2 of the ADC")
+    ch3 = Cpt(EpicsSignal, ":CH3", kind="normal", doc="Input 3 of the ADC")
+    config = Cpt(EpicsSignal, ":CONFIG", kind="config",
+                 doc="Write the current config to the QADC")
+    start = Cpt(EpicsSignal, ":START", kind="normal",
```

Comment:
Should these two have the `command-proc` annotation?

Reply:
Config could, but I think start is best as an enum (running/stopped)

Reply:
Added in this recent commit. Works as expected.